### PR TITLE
fix: Calculating correct gap space when Tooltips have isBeakVisible set to false

### LINKED ISSUE #25927: [Bug] Fluent v8 TooltipHost steals focus from ActionButton when isBeakVisible: false
### Environment Information

- **Package version(s)**: React / v8 (@fluentui/react)
- **Browser and OS versions**: Microsoft Edge 108.0.1462.42

### Describe the issue:

image

When using TooltipHost in conjunction with ActionButton, if `isBeakVisible` is set to false, the tooltip will steal focus from the button when user approaches the button from the bottom (slowly!). Even if `gapSpace` is set to a high value, the tooltip still steals focus.

When reproducing using the codepen below, make sure to slowly hover over the button, starting from the bottom.

#### Actual behavior:

Tooltip steals mouse focus from button.

#### Expected behavior:

Tooltip should not steal the focus from the button.

### If applicable, please provide a codepen repro:

https://codepen.io/anenger/pen/yLEwyoq?editors=1010

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: Consumers of FluentUI, specifically Microsoft Loop

### PR DESCRIPTION
**Pull request by khmakoto**
## Previous Behavior

`Tooltips` with `isBeakVisible` set to `false` were still taking the `beakWidth` into account when calculating the gap space between the `Tooltip` and its `target`, effectively covering part of it sometimes.

Before

## New Behavior

`Tooltips` with `isBeakVisible` set to `false` now calculate the gap space correctly, not taking the `beakWidth` into account for those cases.

After

## Related Issue(s)

- Fixes #25927

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/9bba9cabc4f410930d3acb18cbe5f1d9c867c7f0'>9bba9cabc4f410930d3acb18cbe5f1d9c867c7f0</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-e7f2fdce-71e5-4275-94d7-74ec2b947f78.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-e7f2fdce-71e5-4275-94d7-74ec2b947f78.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27271&commitId=ebb00392e00d61c977e9ece9e810f59932b2f655&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/BaseButton-mount_base.html">990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/BaseButton-mount.html">991</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Breadcrumb-mount_base.html">2374</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Breadcrumb-mount.html">2380</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Checkbox-mount_base.html">2269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Checkbox-mount.html">2216</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/CheckboxBase-mount_base.html">1999</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/CheckboxBase-mount.html">1985</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ChoiceGroup-mount_base.html">3887</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ChoiceGroup-mount.html">3912</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ComboBox-mount_base.html">1061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ComboBox-mount.html">1028</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/CommandBar-mount_base.html">8200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/CommandBar-mount.html">8247</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ContextualMenu-mount_base.html">15513</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ContextualMenu-mount.html">15685</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DefaultButton-mount_base.html">1153</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DefaultButton-mount.html">1156</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DetailsRow-mount_base.html">3062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DetailsRow-mount.html">3068</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DetailsRowFast-mount_base.html">3055</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DetailsRowFast-mount.html">3087</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DetailsRowNoStyles-mount_base.html">2848</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DetailsRowNoStyles-mount.html">2867</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Dialog-mount_base.html">3405</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Dialog-mount.html">3400</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DocumentCardTitle-mount_base.html">450</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/DocumentCardTitle-mount.html">449</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Dropdown-mount_base.html">2751</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Dropdown-mount.html">2747</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/FocusTrapZone-mount_base.html">1677</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/FocusTrapZone-mount.html">1646</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/FocusZone-mount_base.html">1561</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/FocusZone-mount.html">1600</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedList-mount_base.html">51396</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedList-mount.html">60515</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedList-virtual-rerender_base.html">24747</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedList-virtual-rerender.html">24809</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">75543</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedList-virtual-rerender-with-unmount.html">75271</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedListV2-mount_base.html">447</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedListV2-mount.html">434</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedListV2-virtual-rerender_base.html">419</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedListV2-virtual-rerender.html">422</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">443</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">449</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/IconButton-mount_base.html">1612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/IconButton-mount.html">1573</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Label-mount_base.html">566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Label-mount.html">579</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Layer-mount_base.html">3705</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Layer-mount.html">3691</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Link-mount_base.html">655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Link-mount.html">654</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/MenuButton-mount_base.html">1412</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/MenuButton-mount.html">1401</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/MessageBar-mount_base.html">27652</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/MessageBar-mount.html">27732</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Nav-mount_base.html">2727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Nav-mount.html">2691</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/OverflowSet-mount_base.html">1130</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/OverflowSet-mount.html">1134</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Panel-mount_base.html">2161</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Panel-mount.html">2140</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Persona-mount_base.html">1111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Persona-mount.html">1107</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Pivot-mount_base.html">1333</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Pivot-mount.html">1328</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/PrimaryButton-mount_base.html">1235</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/PrimaryButton-mount.html">1273</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Rating-mount_base.html">5791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Rating-mount.html">5878</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/SearchBox-mount_base.html">1253</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/SearchBox-mount.html">1257</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Shimmer-mount_base.html">2446</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Shimmer-mount.html">2408</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Slider-mount_base.html">1805</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Slider-mount.html">1798</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/SpinButton-mount_base.html">3941</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/SpinButton-mount.html">3945</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Spinner-mount_base.html">627</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Spinner-mount.html">646</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/SplitButton-mount_base.html">2644</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/SplitButton-mount.html">2673</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Stack-mount_base.html">674</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Stack-mount.html">671</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/StackWithIntrinsicChildren-mount_base.html">1264</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/StackWithIntrinsicChildren-mount.html">1259</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/StackWithTextChildren-mount_base.html">3394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/StackWithTextChildren-mount.html">3399</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/SwatchColorPicker-mount_base.html">8310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/SwatchColorPicker-mount.html">8394</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/TagPicker-mount_base.html">2179</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/TagPicker-mount.html">2163</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Text-mount_base.html">627</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Text-mount.html">645</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/TextField-mount_base.html">1326</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/TextField-mount.html">1315</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ThemeProvider-mount_base.html">1240</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ThemeProvider-mount.html">1222</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ThemeProvider-virtual-rerender_base.html">882</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ThemeProvider-virtual-rerender.html">890</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1806</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1847</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Toggle-mount_base.html">913</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/Toggle-mount.html">879</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/buttonNative-mount_base.html">406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27271/perf-test/buttonNative-mount.html">396</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/Tooltip/Tooltip.base.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
     this._classNames = getClassNames(styles!, {
       theme: theme!,
       className: className || (calloutProps && calloutProps.className),
-      beakWidth: calloutProps && calloutProps.beakWidth,
+      beakWidth: calloutProps && calloutProps.isBeakVisible && calloutProps.beakWidth ? calloutProps.beakWidth : 0,
```
Comment (sopranopillow) [inline packages/react/src/components/Tooltip/Tooltip.base.tsx:43]: nit:

```suggestion
      beakWidth: calloutProps && calloutProps.isBeakVisible ? calloutProps.beakWidth : 0,
```

**Review by sopranopillow — APPROVED** [review]
minor nit, other than that lgtm

File: packages/react/src/components/Tooltip/Tooltip.base.tsx (hunk 2 above)
Comment (khmakoto) [reply packages/react/src/components/Tooltip/Tooltip.base.tsx]: Done!

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Pull request merged**